.github/workflows/ios.yml

```
@@ -75,13 +75,13 @@ jobs:
   unit-tests:
     name: Unit Tests
-    runs-on: macos-14
+    runs-on: macos-15
 
     steps:
     - uses: actions/checkout@v4
 
-    - name: Select Xcode 16.2
-      run: sudo xcode-select -s /Applications/Xcode_16.2.app
+    - name: Select latest Xcode
+      run: sudo xcode-select -s "$(ls -d /Applications/Xcode*.app | sort -V | tail -1)"
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2012:info:1:25: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)